libs/s25main/figures/nofLeatherWorker.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "nofWorkman.h"
+
+class SerializedGameData;
+class nobUsual;
+
+#ifdef _MSC_VER
+#    pragma warning(disable : 4646) // function declared with [[noreturn]] has non-void return type
+#endif
+
+class nofLeatherWorker : public nofWorkman
+{
+    /// Draw worker at work
+    void DrawWorking(DrawPoint drawPt) override;
+    /// Id in jobs.bob or carrier.bob when carrying a ware
+    [[noreturn]] unsigned short GetCarryID() const override;
+    /// Der Arbeiter erzeugt eine Ware
```

Comment:
I'd not add any more German docstrings. Here it doesn't even add more information than the name of the function already does, so just remove
```suggestion
```

Reply:
done